package.json

```diff
@@ -73,5 +73,8 @@
     "webpack-cli": "3.3.4",
     "webpack-dev-server": "3.8.0",
     "worker-plugin": "3.1.0"
+  },
+  "dependencies": {
+    "wasm-feature-detect": "^1.2.6"
```

Comment:
move this to `devDependencies`

Reply:
How so? This is a runtime dependency.

Reply:
This is off the back of a longer debate, but we have all our dependencies in `devDependencies`.

Reply:
> This is a runtime dependency.

Is buildtime also runtime?

Reply:
> Is buildtime also runtime?

I'd suggest not, at least I prefer to separate "this is a build tool / a plugin for the build tool" vs "this is a runtime library that could as well be requested from external storage".

But I don't really want to get into a longer debate atm, happy to update to follow the existing precedent.

Reply:
Done.